docs/application/web/guides/sensors/ham.md

```diff
@@ -423,12 +365,10 @@ The following table introduces the available monitor types and lists the monitor
 | Monitor                              | Capability                               | Notes                                    |
 | ------------------------------------ | ---------------------------------------- | ---------------------------------------- |
 | Pedometer and accumulative pedometer | `http://tizen.org/feature/sensor.pedometer` | When the pedometer sensor is started, a change callback is invoked when data changes. Use the `getHumanActivityData()` method to get the current data.<br> The accumulative pedometer sensor does not have to be started by your application as long as step counting is enabled by any other application or the system. Listener registered with the `setAccumulativePedometerListener()` method is called when accumulative counters are changed. |
-| Wrist up                             | `http://tizen.org/feature/sensor.wrist_up` | The wrist up sensor is notified when the relevant gesture is performed. The sensor must be enabled using the `start()` method. An event listener invoked when the gesture is detected. This sensor does not provide any data. |
 | Heart rate monitor                   | `http://tizen.org/feature/sensor.heart_rate_monitor` | When the heart rate monitor (HRM) sensor is started, a change callback is invoked when data changes. Use the `getHumanActivityData()` method to get the current data. |
 | GPS                                  | `http://tizen.org/feature/location.batch` | When the GPS sensor is started, a change callback is invoked when data changes. Use the `getHumanActivityData()` method to get the current data.<br> The GPS sensor provides both the current value and a short history of last recorded GPS positions. The sensor supports sampling intervals, which can be used to create more power-efficient applications. |
 | Sleep monitor                        | `http://tizen.org/feature/sensor.sleep_monitor` | When the sleep sensor is started, a change callback is invoked when data changes. Use the `getHumanActivityData()` method to get the current data. |
 | Sleep detector                        | `http://tizen.org/feature/sensor.sleep_monitor` | When the sleep sensor is started, a change callback is invoked when data changes. Use the `getHumanActivityData()` method to get the current data. |
-| Activity recognition                 | `http://tizen.org/feature/sensor.activity_recognition` | To recognize an activity, start listening for it using the `addActivityRecognitionListener()` method.  The following activity types can be recognized:<br> - `STATIONARY`<br> - `WALKING`<br> - `RUNNING`<br> - `IN_VEHICLE` |
```

Comment:
Just a question. In web/tutorials/app-filtering.md file, we list up the features in a table.

In that table `http://tizen.org/feature/sensor.wrist_up` is included. Don't we need to delete that?